appcelerator/titanium_mobile
# fix(ios): hyperloop defineClass() crash

**JIRA:**
https://jira.appcelerator.org/browse/TIMOB-28458

**Summary:**
- Regression as of Titanium 10.0.0.
- Caused by removal of `KrollBridge.require()` method require by hyperloop [here](https://github.com/appcelerator/hyperloop.next/blob/master/iphone/titanium/HyperloopModule.m#L310).
- Also, the `global.require` function was removed in 10.0.0 which is needed by our core JS code [here](https://github.com/appcelerator/titanium_mobile/blob/master/common/Resources/ti.internal/extensions/binding.js#L36).

**Test:**
1. Build and run [hyperloop-examples](https://github.com/appcelerator/hyperloop-examples) for iOS.
2. Go to every window in the app and verify they were as expected.
